<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtMgr</class>
 <widget class="QDialog" name="dtMgr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dreamy Theater Edit Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_entry" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="2" rowspan="2" colspan="2">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox_enable">
           <property name="text">
            <string>Enable entry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_title"/>
         </item>
         <item>
          <widget class="QLabel" name="label_author">
           <property name="text">
            <string>Author:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_author"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="pushButton_apply">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="7">
       <widget class="QGroupBox" name="groupBox_moduleA">
        <property name="title">
         <string>Module A</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="glmodule" name="openGLWidget">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_moduleA">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_characterA"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_moduleA"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QGroupBox" name="groupBox_editdata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Edit data</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_import">
           <property name="text">
            <string>Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_export">
           <property name="text">
            <string>Export...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="7">
       <widget class="QGroupBox" name="groupBox_moduleB">
        <property name="title">
         <string>Module B</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox_moduleB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="glmodule" name="openGLWidget_2">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>190</width>
             <height>304</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_characterB">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_moduleB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_moduleB">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QGroupBox" name="groupBox_audio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Audio</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="0" colspan="3">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButton_adx">
           <property name="text">
            <string>ADX (hack)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="radioButton_builtin">
           <property name="text">
            <string>Built-in:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QRadioButton" name="radioButton_Vorbis">
           <property name="text">
            <string>Ogg Vorbis (hack; 48000 Hz only)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_builtinSong">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_aix">
           <property name="text">
            <string>AIX (hack)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QRadioButton" name="radioButton_MP3">
           <property name="text">
            <string>MPEG-1/2/2.5 Audio Layer III</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_none">
           <property name="text">
            <string>None</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QWidget" name="widget_audiofile" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_audio_preview">
              <property name="text">
               <string>Preview</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_audio_export">
              <property name="text">
               <string>Export...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>Delete entry</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_reset_metadata">
        <property name="text">
         <string>Reset metadata</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glmodule</class>
   <extends>QOpenGLWidget</extends>
   <header>glmodule.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
